paywall/src/stories/lock/Lock.stories.js

```diff
@@ -5,11 +5,22 @@ import { Lock } from '../../components/lock/Lock'
 import createUnlockStore from '../../createUnlockStore'
 import { ConfigContext } from '../../utils/withConfig'
 import { UNLIMITED_KEYS_COUNT } from '../../constants'
+import { WindowContext } from '../../hooks/browser/useWindow'
```

Comment:
I am confused by the choice of variable name. It looks like most (all?) hooks use `useXX`. why here a name which looks like a component?

Reply:
But that is probably for another PR: maybe add a `TODO rename` comment so we can keep track of this?

Reply:
this is the context, defined with `React.createContext` that is used to pass the `window` down. `useWindow.js` exports both this and the hook `useWindow`.